packages/orm/src/drivers/buildKnexQuery.ts

```diff
@@ -67,7 +66,10 @@ export function buildKnexQuery(
       query.orderBy(`${alias}.${column}`, order);
     });
 
-  query.limit(limit || entityLimit);
```

Comment:
Yeah it was intentiona. There's no way to know which `em` will eventually get use the generated knex query.  If we used the default it might be higher or lower than what we eventually want.  `executeFind` will pass an explicit limit from the `em`.

Reply:
Nice, cool `executeFind` passing the EM's value...maybe pedantically we still have code that calls directly into `buildKnexQuery`, so could probably use the default/global entity limit as a fallback for those callers?

Reply:
We only have 2 queries generated by `buildQuery` that don't eventually go through `loadFromQuery`.  One of them applies its own limit, and the other eventually goes through `loadFromRows`.  So, in every case where we manually build a knex query the code would still fail, it'd just take slightly longer for 50k entities to be created in memory.

Reply:
Sounds good, fwiw still doesn't really sound like there is a reason to _not_ put this here, but :shrug: won't block on it.

Reply:
it'd effectively override the `em`'s limit on `loadFromQuery` calls